Cartfile

```diff
@@ -1,2 +1,2 @@
-github "pinterest/PINRemoteImage" "3.0.0-beta.6"
+github "pinterest/PINRemoteImage" "master"
```

Comment:
Yes. I found out that necessary headers was added in `3.0.0-beta.7`, but unfortunately I wasn't able to build it via Carthage. Also there are these lines of code that enables support of animated images 
```
#if __has_include (<PINRemoteImage/PINAnimatedImage.h>)
#define PIN_ANIMATED_AVAILABLE 1
#import <PINRemoteImage/PINAnimatedImage.h>
#import <PINRemoteImage/NSData+ImageDetectors.h>
#import <PINRemoteImage/PINAlternateRepresentationProvider.h>
#else
#define PIN_ANIMATED_AVAILABLE 0
#endif
```
But these 3 headers aren't `public` in `PINRemoteImage` framework
```
#import <PINRemoteImage/PINAnimatedImage.h>
#import <PINRemoteImage/NSData+ImageDetectors.h>
#import <PINRemoteImage/PINAlternateRepresentationProvider.h>
```
so if you make them `public` we will get support of animated images via Carthage too.